apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -48,6 +49,16 @@ const CurriculumCatalog = ({
     setAssignSuccessMessage('');
   };
 
+  const getSelfPacedPlPath = selfPacedPlId => {
+    if (selfPacedPlId !== null) {
+      const course_offering = selfPacedPlCourseOfferings.find(
+        curriculum => curriculum.id === selfPacedPlId
+      );
+      return course_offering ? course_offering.course_version_path : null;
```

Comment:
Nit: if we do keep this function, this can be `return course_offering?.course_version_path;`